cmd/protoc-gen-goclay/swagger.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/golang/glog"
 	plugin "github.com/golang/protobuf/protoc-gen-go/plugin"
 	"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
-	"github.com/utrack/grpc-gateway/protoc-gen-swagger/genswagger"
+	"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/genswagger"
```

Comment:
Hmm, then the swagger would break if `clay` consumers won't use `github.com/utrack/grpc-gateway` explicitly.
Maybe it's better to keep it as it is.